packages/workbox-webpack-plugin/src/index.js

```diff
@@ -23,9 +23,9 @@ const copyWorkboxSW = require('./lib/utils/copy-workbox-sw');
 const {setReadFile} = require('./lib/utils/read-file');
 
 /**
- * Use the instance of this in the plugins array of the webpack config.
+ * Use the instance of this in the `plugins` array of the Webpack config.
  *
- * @example <caption>Zero-configuration</caption>
+ * @example <caption>Zero-configuration usage:</caption>
```

Comment:
Can we remove these examples and cover them in the webpack module docs: 

https://docs.google.com/document/d/1w53nZZCAT765IE3eAzbMg1qP4z72BJ3Ors3vz6QQP4A/edit?usp=sharing